Kuroba/app/src/main/res/values/strings.xml

```diff
@@ -145,33 +145,33 @@ Re-enable this permission in the app settings if you permanently disabled it."</
     <string name="open_link_confirmation">Open this link?</string>
     <string name="open_thread_confirmation">Open this thread?</string>
     <string name="open_archived_thread_confirmation">Open this archived thread?</string>
-    <string name="open_link_failed">No applications were found to open this link</string>
-    <string name="open_file_picker_failed">No file picker applications were found</string>
+    <string name="open_link_failed">Install an app that can open this link first.</string>
+    <string name="open_file_picker_failed">Install a file-picker app first.</string>
```

Comment:
Not finding an application and not having one installed can be different things. Some apps are dumb, don't instantly assume one isn't installed because Kuroba couldn't find it.

Reply:
True, I'll change to something reflecting not being able to open files.

Reply:
Went with a "not assumed dumb" option.